products/sourcegraph.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Sourcegraph
+category: server-app
+tags: java-runtime
+permalink: /sourcegraph
+releasePolicyLink: https://handbook.sourcegraph.com/departments/engineering/dev/process/releases/
+releaseDateColumn: true
+eolColumn: Support
+
+identifiers:
+- purl: pkg:github/sourcegraph/sourcegraph
+- purl: pkg:docker/sourcegraph/sg
+
+auto:
+  methods:
+  - git: https://github.com/sourcegraph/sourcegraph.git
+
+releases:
+- releaseCycle: "5"
+  releaseDate: 2023-05-22
+  eol: false
+  latest: "5.3.3"
+  latestReleaseDate: 2024-03-20
+  link: https://github.com/sourcegraph/sourcegraph/releases/tag/v__LATEST__
+
+- releaseCycle: "4"
+  releaseDate: 2022-09-22
+  eol: 2023-05-22
+  latest: "4.5.1"
+  latestReleaseDate: 2023-02-24
+  link: https://sourcegraph.com/blog/release/4.5
+
+---
+
+> [Sourcegraph](https://sourcegraph.com/) is a code search and intelligence tool for developers that helps
+> them find, understand, and fix code across all of their projects and dependencies.
+
+Sourcegraph is available in self-hosted and cloud editions. This page only tracks releases for
+the self-hosted offering.
+
+Looking at recent releases, it seems that only the latest version of each major release is supported.
```

Comment:
```suggestion
Looking at recent releases, it seems that only the latest release is supported.
```

Because you set the EOL date for 4.x to the release date of 5.x.